spec/event_builder_spec.rb

```diff
@@ -34,216 +34,336 @@
 
     time_now = Time.now
     allow(Time).to receive(:now).and_return(time_now)
+    allow(SecureRandom).to receive(:uuid).and_return('a68cf1ad-0393-4e18-af87-efe8f01a7c9c');
```

Comment:
What's the need of it?

Reply:
To generate expected payload for Testing purpose